### ISSUE TITLE
Question regarding validation script during fine-tuning

### ISSUE THREAD

**Issue by gurunathparasaram** [grammatical/magec-wnut2019#1]
As described in this [paper](https://www.aclweb.org/anthology/W19-4427.pdf), I completed pre-training for the low-resource model using 100 million sentences using Marian(without edit-weighting) and got an F-score of 25 on the WI-LOCNESS dev-set (Experiments in the paper achieved around 26.7 F-score). While fine-tuning(I use edit-weighting for fine-tuning), I observe that the validation score for the `translation` metric obtained from the validation script starts at around 4 F-score, but increases slowly with training, on WI-LOCNESS dev-set, whereas the same model.npz when used for inference using `marian-decoder`on WI-dev set using the command:
 `cat devset.err | ../../marian-dev/build/marian-decoder -c config.yml -d 1  --quiet-translation -o $OP_FILE`
yields an F-score of 25-27. The config can be found here: [config.log](https://github.com/grammatical/magec-wnut2019/files/4019003/config.log)
 - I observe that the F-scores from the `translation` metric obtained via validate.sh and that from doing inference using `marian-decoder` command are in different range altogether. Why do the scores vary? Is this expected?
 - I checked the outputs generated by the model (before and after remove_repetitions.py) while using the validation script in devset.out and they seem to differ from those generated while using `marian-decoder`. ~~Will using `marian-decoder` command mentioned above, in the validation script, help me achieve the 38 F-score on the WI-LOCNESS dev-set after fine-tuning as achieved in the paper?~~
 - Any other suggestions to fine-tune the model? Thanks in advance.
